content/docs/command-reference/gc.md

```diff
@@ -5,7 +5,7 @@ Remove unused files and directories from <abbr>cache</abbr> or [remote storage].
 ## Synopsis
 
 ```usage
-usage: dvc gc [-h] [-q | -v] [-w] [-a] [-T] [--all-commits]
+usage: dvc gc [-h] [-q | -v] [-w] [-a] [-T] [--all-commits] [--date <commit_date>]
               [--all-experiments] [-c] [-r <name>] [-f] [-j <number>]
               [-p [<path> [<path> ...]]]
```

Comment:
Remember that these blocks scroll easily 🙂 -- it's best to quickly [check the review app](https://dvc-org-time-flag-for-g-ny2vdc.herokuapp.com/doc/command-reference/gc) when making any changes:

![image](https://user-images.githubusercontent.com/1477535/178380814-9a334731-c6c5-4433-a05e-13ca2835e1a1.png)

Reply:
Fix (committing):

```suggestion
usage: dvc gc [-h] [-q | -v] [-w] [-a] [-T] [--all-commits]
              [--date <commit_date>] [--all-experiments]
              [-c] [-r <name>] [-f] [-j <number>]
              [-p [<path> [<path> ...]]]
```